pysat/_instrument.py

```diff
@@ -1890,7 +1890,11 @@ def concat_data(self, new_data, prepend=False, **kwargs):
         else:
             # Specify the dimension, if not otherwise specified
             if 'dim' not in kwargs:
-                kwargs['dim'] = self.index.name
+                if 'time' in self.data.indexes:
+                    kwargs['dim'] = 'time'
+                elif 'Epoch' in self.data.indexes:
+                    kwargs['dim'] = 'Epoch'
+
```

Comment:
Following conventions elsewhere in _instrument.  The change of the behaviour of `index` in the test instruments broke this